gnome-extra/krb5-auth-dialog/krb5-auth-dialog-43.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson gnome2-utils
+
+DESCRIPTION="An application to acquire and list Kerberos tickets."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/krb5-auth-dialog/"
+SRC_URI="https://download.gnome.org/sources/${PN}/$(ver_cut 1)/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=gui-libs/libadwaita-1.2.0:1
```

Comment:
The depends should be sorted too

Reply:
Also please check if `gui-libs/libadwaita` really is needed

Reply:
right that `gui-libs/libadwaita` only comes with 44.alpha1 which i do not yet propose here

Reply:
hope i did get that sort order right